Resources/config/extractor_relations.yml

```diff
@@ -1,6 +1,5 @@
 parameters:
     # Sql relations classes
-    ongr_connections.sql_relations.simple_relation.class: ONGR\ConnectionsBundle\Sync\Extractor\Relation\SimpleSqlRelation
-    ongr_connections.sql_relations.composed_relation.class: ONGR\ConnectionsBundle\Sync\Extractor\Relation\ComposedSqlRelation
+    ongr_connections.sql_relations.sql_relation.class: ONGR\ConnectionsBundle\Sync\Extractor\Relation\SqlRelation
```

Comment:
`ongr_connections.sql_relations.sql_relation.class` is used by relations example https://github.com/ongr-io/ConnectionsBundle/pull/143/files#diff-0ecea2bd068f83cc8f0d36e45f499f83R33